plugins/uartdmx/README.md

```diff
@@ -0,0 +1,34 @@
+Native UART DMX Plugin
```

Comment:
As discussed, please remove this file and edit PluginDescription if targeting 0.10.